packages/user-service/services/users/user.js

```diff
@@ -6,7 +6,7 @@ const { captureAWSv3Client } = require('aws-xray-sdk-core')
 
 const log = require('slic-tools/log')
 
-const cognitoCore = new CognitoIdentityProviderClient({ region: process.env.AWS_REGION })
```

Comment:
the region will be automatically be picked up using the lambda environment variables